docs/global/modules/samples/pages/use_case_2.adoc

```diff
@@ -28,53 +32,60 @@ image::use_case_2_sequence.svg[Static,1920]
 // Mermaid source. Edit with https://mermaid.live/
 // sequenceDiagram
 //     actor back-office
-//     actor tutor
 //     actor student
+//     actor tutor
 //     tutor->>Management application: make change
 //     Management application-->>tutor: display change
-//     Management application->>Model server: communicate change
+//     Management application->>Model server: propagate change
 //     Model server-->>Management application:
-//     Model server->>Overview application: communicate change
+//     Model server->>Overview application: propagate change
 //     Overview application-->>Model server:
 //     Overview application-->>student: display change
-//     Model server->>MPS: communicate change
+//     Model server->>MPS: propagate change
 //     MPS-->>Model server:
 //     MPS-->>back-office: display change
 //     back-office->>MPS: make change
 //     MPS-->>back-office: display change
-//     MPS->>Model server: communicate change
+//     MPS->>Model server: propagate change
 //     Model server-->>MPS:
-//     Model server->>Overview application: communicate change
+//     Model server->>Overview application: propagate change
 //     Overview application-->>Model server:
 //     Overview application-->>student: display change
-//     Model server->>Management application: communicate change
+//     Model server->>Management application: propagate change
 //     Management application-->>Model server:
 //     Management application-->>tutor: display change
 
-== Build
+In this demonstration
+
+1. Model data is edited through the *Management application*.
+2. Those changes are then synced to the *Overview application* and MPS through the model server.
+3. Model data is edited again in MPS by reverting the made changes.
+4. Those changes are then also synced back.
+
+== Instructions
+
+You can try the demo yourself by downloading the https://github.com/modelix/modelix.samples/[samples repository].
```

Comment:
> We could add a short sentence before "Build and Start", something like: "You can try the demo yourself, just follow these steps:"

> Where do we find the spa-management-vue and mps projects? It might be good to link the samples repository as a first step and ask the user to clone it locally.